src/app/src/test/kotlin/org/imperial/mrc/hint/integration/ADRTests.kt

```diff
@@ -109,39 +109,38 @@ class ADRTests : SecureIntegrationTests()
             assertThat(data["id"].textValue()).isEqualTo(id)
         }
     }
-
-    @ParameterizedTest
-    @EnumSource(IsAuthorized::class)
-    fun `can save pjnz from ADR`(isAuthorized: IsAuthorized)
-    {
-        val pjnz = extractUrl(isAuthorized, "inputs-unaids-spectrum-file")
-        val result = testRestTemplate.postForEntity<String>("/adr/pjnz",
-                getPostEntityWithUrl(pjnz))
-        assertSecureWithSuccess(isAuthorized, result, "ValidateInputResponse")
-    }
-
-    @ParameterizedTest
-    @EnumSource(IsAuthorized::class)
-    fun `can save population from ADR`(isAuthorized: IsAuthorized)
-    {
-        val population = extractUrl(isAuthorized, "inputs-unaids-population")
-        val result = testRestTemplate.postForEntity<String>("/adr/population",
-                getPostEntityWithUrl(population))
-        assertSecureWithSuccess(isAuthorized, result, "ValidateInputResponse")
-    }
-
-    @ParameterizedTest
-    @EnumSource(IsAuthorized::class)
-    fun `can save shape from ADR`(isAuthorized: IsAuthorized)
-    {
-        val shape = extractUrl(isAuthorized, "inputs-unaids-geographic")
-        val result = testRestTemplate.postForEntity<String>("/adr/shape",
-                getPostEntityWithUrl(shape))
-        assertSecureWithSuccess(isAuthorized, result, "ValidateInputResponse")
-    }
-
+//
```

Comment:
Looks like all adr files are being intermittently deleted (possibly while demo-ing the app during the unaids ref group meeting this afternoon) so have had to comment out all these tests for now.